docs/reference/hooks.md

```diff
@@ -379,6 +379,25 @@ Return a QuerySet of `Permission` objects to be shown in the Groups administrati
       ])
 ```
 
+(register_user_listing_buttons)=
+
+### `register_user_listing_buttons`
+
+Add buttons to the user list. This example will add a simple button to the listing:
+
+```python
+from wagtail.admin import widgets as wagtailadmin_widgets
+
+@hooks.register("register_user_listing_buttons")
+def user_listing_external_profile(context, user):
+    yield wagtailadmin_widgets.Button(
```

Comment:
suggestion: use [`UserListingButton`](https://github.com/wagtail/wagtail/blob/535f6e574ba8516de68188b997a6efc8c54266cb/wagtail/users/widgets.py#L4)

for consistency with https://github.com/wagtail/wagtail/blob/535f6e574ba8516de68188b997a6efc8c54266cb/wagtail/users/wagtail_hooks.py#L139

Otherwise LGTM

Reply:
Done locally, will push & merge thanks!